share/lib/python/neuron/rxd/generalizedReaction.py

```diff
@@ -141,6 +141,7 @@ def _update_indices(self):
                 if reg._secs1d or reg._secs3d:
                     break
             else:
+                if not sources_ecs or not dests_ecs:
```

Comment:
so if sp_regions, then everything after this line depends on there being both sources_ecs and dests_ecs?